docs/content/guide/filter.ngdoc

```diff
@@ -31,6 +31,16 @@ Filters may have arguments. The syntax for this is
 E.g. the markup `{{ 1234 | number:2 }}` formats the number 1234 with 2 decimal points using the
 {@link ng.filter:number `number`} filter. The resulting value is `1,234.00`.
 
+### When filters are executed
+
+In templates, filters are only executed when their inputs have changed. This is more performant than executing
```

Comment:
I would clarify that this only applies to stateless filters (the default kind). Filters marked as stateful are called on every digest.